pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go

```diff
@@ -278,3 +278,11 @@ func admissionFlags(admissionPluginConfig map[string]configv1.AdmissionPluginCon
 
 	return args, nil
 }
+
+func sniCertKeys(namedCertificates []configv1.NamedCertificate) []string {
+	args := []string{}
+	for _, nc := range namedCertificates {
+		args = append(args, fmt.Sprintf("%s,%s:%s", nc.CertFile, nc.KeyFile, strings.Join(nc.Names, ",")))
```

Comment:
have you checked that we support `foo,bar:` without the names?

Reply:
Yes, it'll parse this as empty, see here: https://github.com/openshift/origin/blob/2b961cd33770c2593dc31e9d1d2a73bdd70da631/vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/util/flag/namedcertkey_flag.go#L44-L53